uber/site_sections/registration.py

```diff
@@ -165,15 +164,9 @@ def qrcode_generator(self, data):
         this function.  Or, offload image generation to a dedicated microservice that replicates this functionality.
 
         """
-        if not c.QR_CODE_PASSWORD or not qr_cipher:
-            # Don't generate a 2D barcode if we aren't configured for encryption
-            return '2D barcode password not set, cannot generate images'
-
-        encrypted_data = c.EVENT_QR_ID + bytes.decode(binascii.hexlify(qr_cipher.encrypt(uuid.UUID(data).hex)))
-
         checkin_barcode = treepoem.generate_barcode(
             barcode_type='azteccode',
-            data=encrypted_data,
+            data=c.EVENT_QR_ID + str(data),
```

Comment:
We're still adding the EVENT_QR_ID for safety reasons -- even though UUIDs are designed to have no collision, it costs us very little add and strip this ID for a guarantee of no collisions between events ever.